package/suse-migration-pre-checks-spec-template

```diff
@@ -0,0 +1,62 @@
+#
+# spec file for package suse-migration-pre-checks
+#
+# Copyright (c) 20022 SUSE LINUX GmbH, Nuernberg, Germany.
```

Comment:
I had no idea I lived that long already ;) (extra 0 in the year)

Reply:
Thanks, fixed